### ISSUE TITLE
CSV result format - V1.1 Request

### ISSUE THREAD

**Issue by KathiSchleidt** [INSIDE-information-systems/SensorThingsAPI#3]
In order to make STA data more accessible, it would be valuable to provide a simple CSV result format

Link Fraunhofer GitHub: https://github.com/FraunhoferIOSB/FROST-Server/issues/58
Link OGC GitHub: https://github.com/opengeospatial/sensorthings/issues/51

**Comment by liangsteve** [comment]
I totally agree that it's very useful to have CSV outputs.  Can you please offer more details here?  There are many many different ways to flatten the data model and provide CSV outputs.  Thoughts?

**Comment by sgrellet** [comment]
Proposal from our workshop at BRGM with Hylke and Kathi and BRGM folks
Rationale only traverse the single cardinalities
* mainly feasible for Observation and Datastream
  * Observation : start from Observation, traverse the single cardinalities (no funky loops) and write everything flat at the result level. Will only miss the Location
  * Datastream will have Thing, Sensor, ObservedProperty but no Observation nor Location
* for the others, its just a csv 'dump' of the single table
* MultiDatastream might a bit tricky

**Comment by hylkevds** [comment]
When expanding over a "many" relation, we can just return multiple lines, partially duplicating data. These queries should return equivalent results:

- v1.0/Observations?$expand=Datastreams
- v1.0/Datastreams?$expand=Observations

The only question would be what to do with a Datastream with no Observations. I think it would be best to return a single, incomplete line.
